non-critical-infra/packages/encrypt-email-address/encrypt-email-address.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+import re
+import subprocess
+from pathlib import Path
+from textwrap import dedent
+
+import click
+
+
+def find_project_root(start: Path) -> Path:
+    # Can search for `flake.nix` because there are multiple in this project.
+    root_indicator = start / ".git/config"
+    if root_indicator.exists():
+        return start
+
+    return find_project_root(start.parent)
+
+
+@click.command(
+    help=dedent("""\
+        Encrypt an email address (or email addresses) for inclusion in a mailing list.
+
+        See `non-critical-infra/modules/mailserver/mailing-lists.nix` for
+        documentation about how to use this.
+    """)
+)
+@click.argument("user")
+@click.argument("email")
+@click.option("--force/--no-force", "-f/ ", default=False)
+def main(user: str, email: str, force: bool) -> None:
```

Comment:
@Mic92, I don't think "user" is really the right name for this. Perhaps `address_id`?

Reply:
Done!